src/main/java/com/exactpro/th2/sailfish/utils/MathFilter.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.th2.sailfish.utils;
+
+import java.math.BigDecimal;
+import java.text.NumberFormat;
+import java.text.ParseException;
+
+import com.exactpro.sf.aml.scriptutil.ExpressionResult;
+import com.exactpro.sf.aml.scriptutil.StaticUtil.IFilter;
+import com.exactpro.th2.common.grpc.FilterOperation;
+
+public class MathFilter implements IFilter {
+
+    private final Number value;
+    private final String stringFormatOperation;
+    private final FilterOperation operation;
+
+    public MathFilter(FilterOperation operation, String value) throws ParseException {
+        this.value = NumberFormat.getInstance().parse(value);
```

Comment:
Are you sure that the `NumberFormat` will normally handle the value when a different local is used in the machine? I mean the floating-point number can use a dot or a comma as a separator (depending on the locale). We must be sure that we always expect the value with a dot